mvn_setup/tasks-parent/tasks-ui/src/core/TaskEdit/TaskEditDialog.tsx

```diff
@@ -3,9 +3,9 @@ import { Box, Stack, List, Dialog, Button, Grid } from '@mui/material';
 import SecurityIcon from '@mui/icons-material/Security';
 import { FormattedMessage } from 'react-intl';
 import TaskClient from '@taskclient';
-import { DatePicker } from '../../DatePicker/DatePicker';
-import { StyledFullScreenDialog } from '../StyledFullScreenDialog';
-import Fields from './EditTaskFields';
+import { DatePicker } from '../DatePicker/DatePicker';
+import { StyledFullScreenDialog } from '../Dialogs/StyledFullScreenDialog';
```

Comment:
We should make an `index.ts` inside of the Dialogs folder. It should export the StyledFullScreenDialog as default export, and then this import should be fixed to import from the index instead of the individual component.